cmd/agola/cmd/projectupdate.go

```diff
@@ -51,6 +52,7 @@ func init() {
 	flags.StringVarP(&projectUpdateOpts.name, "name", "n", "", "project name")
 	flags.StringVar(&projectUpdateOpts.parentPath, "parent", "", `parent project group path (i.e "org/org01" for root project group in org01, "user/user01/group01/subgroub01") or project group id where the project should be moved`)
 	flags.StringVar(&projectUpdateOpts.visibility, "visibility", "public", `project visibility (public or private)`)
+	flags.BoolVar(&projectUpdateOpts.passVarsToForkedPR, "pass-vars-to-forked-pr", false, `pass variables to run even if triggered by PR from forked repo (true or false)`)
```

Comment:
I'll make the option comment the same for this and project create. I'll just remove the part inside the brakets since the default and the type are automatically provided by cobra.